opencti-platform/opencti-graphql/src/graphql/sseMiddleware.js

```diff
@@ -281,17 +287,38 @@ const createSeeMiddleware = () => {
       res.status(500).end();
     }
   };
+  const isFoundsTrue = async (founds, filterMode) => {
```

Comment:
I think we should do a quick pair on this, I don't understand the need at all

Reply:
fixed